Description: fix manpage hyphen-used-as-minus-sign errors
Author: Julien Valroff <julien@debian.org>
Last-Updated: 2011-12-30

--- a/doc/gsimplecal.1
+++ b/doc/gsimplecal.1
@@ -27,11 +27,11 @@
 
 .SH COMMANDS AND OPTIONS
 .TP 5
-\fB-v, --version\fP
+\fB-v, -\-version\fP
 Print the program name and version to stdout, then exit with code 0.
 
 .TP 5
-\fB-h, --help\fP
+\fB-h, -\-help\fP
 Print the short usage help to stderr, then exit with error code 2.
 
 .TP 5
@@ -63,7 +63,7 @@
 .br
 show_week_numbers = 0
 .br
-external_viewer = sunbird -showdate "%Y-%m-%d"
+external_viewer = sunbird \-showdate "%Y\-%m\-%d"
 .br
 clock_format = %a %d %b %H:%M
 .br
